Leave/leave_interface.py

```diff
@@ -154,6 +154,23 @@ def GetReasonOfLeaves(leaves_array):
 
     return leaves_array[0]["reason"]
 
+def GetMissingMembersByRole(members_all, role, voice_channel_members):
+
+    date = datetime.datetime.strptime(datetime.datetime.today().strftime('%d/%m/%Y'), '%d/%m/%Y')
+
+    role_member_list = list(filter(lambda member: role in member.roles, members_all))
+    role_member_in_voicechannel_list = list(filter(lambda member : role in member.roles, voice_channel_members))
```

Comment:
```suggestion
    role_members_in_voicechannel = list(filter(lambda member : role in member.roles, voice_channel_members))
```
Plurality + the word `list` is redundant. We want descriptive names while avoiding redundancy.

Reply:
Changed variables names to be appropriately descriptive